tests/bluetooth/tester/src/mesh.c

```diff
@@ -741,53 +799,1437 @@ static void proxy_identity_enable(uint8_t *data, uint16_t len)
 		   err ? BTP_STATUS_FAILED : BTP_STATUS_SUCCESS);
 }
 
-void tester_handle_mesh(uint8_t opcode, uint8_t index, uint8_t *data, uint16_t len)
+static void composition_data_get(uint8_t *data, uint16_t len)
 {
-	switch (opcode) {
-	case MESH_READ_SUPPORTED_COMMANDS:
-		supported_commands(data, len);
-		break;
-	case MESH_CONFIG_PROVISIONING:
-		config_prov(data, len);
-		break;
-	case MESH_PROVISION_NODE:
-		provision_node(data, len);
-		break;
-	case MESH_INIT:
-		init(data, len);
-		break;
-	case MESH_RESET:
-		reset(data, len);
-		break;
-	case MESH_INPUT_NUMBER:
-		input_number(data, len);
-		break;
-	case MESH_INPUT_STRING:
-		input_string(data, len);
-		break;
-	case MESH_IVU_TEST_MODE:
-		ivu_test_mode(data, len);
-		break;
-	case MESH_IVU_TOGGLE_STATE:
-		ivu_toggle_state(data, len);
-		break;
-	case MESH_LPN:
-		lpn(data, len);
-		break;
-	case MESH_LPN_POLL:
-		lpn_poll(data, len);
-		break;
-	case MESH_NET_SEND:
-		net_send(data, len);
-		break;
-	case MESH_HEALTH_GENERATE_FAULTS:
-		health_generate_faults(data, len);
-		break;
-	case MESH_HEALTH_CLEAR_FAULTS:
-		health_clear_faults(data, len);
-		break;
-	case MESH_MODEL_SEND:
-		model_send(data, len);
+	struct mesh_comp_data_get_cmd *cmd = (void *)data;
+	uint8_t page;
+	struct net_buf_simple *comp = NET_BUF_SIMPLE(14);
+	int err;
+
+	LOG_DBG("");
+
+	net_buf_simple_init(comp, 0);
+
+	err = bt_mesh_cfg_comp_data_get(cmd->net_idx, cmd->address, cmd->page,
+					&page, comp);
+	if (err) {
+		LOG_ERR("err %d", err);
+		goto fail;
+	}
+
```

Comment:
Send composition data via `tester_send` to the automation framework. It can be used in other parts of testing.